platform/src/main/java/platform/pageobjects/Footer/TermsConditionsFooter.java

```diff
@@ -0,0 +1,371 @@
+	public TermsConditionsFooter(WebDriver driver) {
+		this.driver = driver;
+		PageFactory.initElements(driver, this);
+	}
+
+	public void switchToPreviousTab() {
+		ArrayList<String> tab = new ArrayList<>(driver.getWindowHandles());
+		driver.switchTo().window(tab.get(1));
+		driver.close();
+		driver.switchTo().window(tab.get(0));
+		// driver.close();
+	}
+
+	@FindBy(xpath = "//div[@class='container container--small']//div[3]//div[1]//a[1]")
+	public WebElement casecTermsLink;
+
+	@FindBy(xpath = "//div[@class='container container--small']//div[3]//div[1]//a[2]")
+	public WebElement jyukuPitaTermsLink;
+
+	@FindBy(xpath = "//div[@class='container container--small']//div[3]//div[1]//a[3]")
+	public WebElement jyukenTermsLink;
+
+	@FindBy(xpath = "//div[@class='container container--small']//div[3]//div[1]//a[4]")
+	public WebElement consultTermsLink;
+
+	@FindBy(xpath = "//div[@class='container container--small']//div[3]//div[1]//a[5]")
+	public WebElement EikenTermsPDFLink;
+
+	@FindBy(xpath = "//div[@class='container container--small']//div[3]//div[1]//a[6]")
+	public WebElement EikenTermsLink;
+
+	@FindBy(xpath = "//a[normalize-space()='http://www.jiem.co.jp/privacy/index.html']")
+	public WebElement JIEMTermsLink;
+
+	@FindBy(xpath = "//div[@class='container container--small']//div[13]//div[2]//a[1]")
+	public WebElement SupportTermsLink;
+
+	@FindBy(xpath = "//a[normalize-space()='https://www.ei-navi.jp/policy/site_policy.html']")
+	public WebElement einaviPolicyLink;
+
+	@FindBy(xpath = "//div[@class='ep-term__content']//div[15]//div[1]/a[1]")
+	public WebElement fbLPolicyLink;
+
+	@FindBy(xpath = "//div[@class='ep-term__content']//div[15]//div[2]/a[1]")
+	public WebElement FacebookOptoutLink;
+
+	@FindBy(xpath = "//div[@class='ep-term__content']//div[15]//div[3]/a[1]")
+	public WebElement TwitterPrvivacyPolicy;
+
+	@FindBy(xpath = "//div[@class='ep-term__content']//div[15]//div[4]/a[1]")
+	public WebElement CustomizedAdLink;
+
+	@FindBy(xpath = "//div[@class='ep-term__content']//div[15]//div[5]/a[1]")
+	public WebElement LinePrivacyPolicyLink;
+
+	@FindBy(xpath = "//div[@class='ep-term__content']//div[15]//div[6]/a[1]")
+	public WebElement LineBehavioralAdvertisingLink;
+
+	@FindBy(xpath = "//div[@class='ep-term__content']//div[15]//div[7]/a[1]")
+	public WebElement AmazonTargetAdTermsLink;
+
+	@FindBy(xpath = "//div[@class='ep-term__content']//div[15]//div[8]/a[1]")
+	public WebElement ChangeDisplaySettingLink;
+
+	@FindBy(xpath = "//div[@class='ep-term__content']//div[15]//div[9]/a[1]")
+	public WebElement BrainPadLink;
+
+	@FindBy(xpath = "//div[@class='ep-term__content']//div[15]//div[10]/a[1]")
+	public WebElement Everrise;
+
+	@FindBy(xpath = "//div[@class='ep-term__content']//div[15]//div[11]/a[1]")
+	public WebElement NetworkAdvertisingLink;
+
+	@FindBy(xpath = "//div[@class='ep-term__content']//div[15]//div[12]/a[1]")
+	public WebElement AboutadsLink;
+
+	@FindBy(xpath = "//div[@class='ep-term__content']//div[15]//div[13]/a[1]")
```

Comment:
Plz replace absolute xpath with relative ones wherever applicable

Reply:
I checked the xpaths for terms and conditions links but they have same class so It when I added those classes xpaths, it was not identifying. So had to take unique class from from webpage and had to add xpath from then on.

Reply:
@csonparote san, I have updated the xpaths.